features/support/helpers.rb

```diff
@@ -110,26 +114,27 @@ def run_in_shell(*args)
 
   p.value
 end
+# rubocop:enable Metrics/AbcSize
 
 #
 
 def slug(title = nil)
   if !title
     then Time.now.strftime("%s%9N") # nanoseconds since the Epoch
-    else title.downcase.gsub(/[^\w]/, " ").strip.gsub(/\s+/, '-')
+  else title.downcase.gsub(/[^\w]/, " ").strip.gsub(/\s+/, "-")
   end
 end
 
 #
```

Comment:
@envygeeks What is the purpose of the `#` separating some methods?
Seems to have been added in #4343 #4347